website/src/components/FAQ/readme.mdx

```diff
@@ -144,15 +113,15 @@ failover.
 
 Service accounts facilitate secure connections from a server, VM or container to
 a Resource. Because service accounts do not involve a user, they can’t fulfill
-2FA requirements, and use the Firezone client in a non-interactive way (headless
+2FA requirements, and use the Firezone Client in a non-interactive way (headless
 mode).
```

Comment:
Github isn't letting me add a suggestion here but

This could parse as "Because services accounts do not involve a user, they can't (fulfill requirements, and use the Firezone Client in a non-interactive way)"

Consider "Because service accounts do not involve a user, they can't (fulfill 2FA requirements), so they (use the Firezone Client in a non-interactive way)"

Or to put the positive first and details second, "Service accounts use the Firezone Client in non-interactive mode ("headless" mode), since they do not involve user interaction and cannot fulfill 2FA requirements."